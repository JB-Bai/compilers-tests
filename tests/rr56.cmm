int;
int nAAhFztw1B4, RElG95AmM, tKw1szk;
struct ivHLcVe{
}
;

