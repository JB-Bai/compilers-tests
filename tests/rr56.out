Program (1)
  ExtDefList (1)
    ExtDef (1)
      Specifier (1)
        TYPE: int
      SEMI
    ExtDefList (2)
      ExtDef (2)
        Specifier (2)
          TYPE: int
        ExtDecList (2)
          VarDec (2)
            ID: nAAhFztw1B4
          COMMA
          ExtDecList (2)
            VarDec (2)
              ID: RElG95AmM
            COMMA
            ExtDecList (2)
              VarDec (2)
                ID: tKw1szk
        SEMI
      ExtDefList (3)
        ExtDef (3)
          Specifier (3)
            StructSpecifier (3)
              STRUCT
              OptTag (3)
                ID: ivHLcVe
              LC
              RC
          SEMI
